packages/react-components/src/new-context-api/LoadingContainer.js

```diff
@@ -0,0 +1,99 @@
+import React, { Children, Component } from "react";
+import PropTypes from "prop-types";
+
+/*
+ * Create component.
+ */
+
+class LoadingContainer extends Component {
+  render() {
+    if (this.props.drizzleState) {
+      if (this.props.drizzleState.web3.status === "failed") {
+        if (this.props.errorComp) {
```

Comment:
As above, this can be written as:

```js
return this.props.errorComp || <main>...</main>
```

In order to avoid another if-statement.